ProcessMessage/handler.ts

```diff
@@ -422,6 +425,23 @@ export const getProcessMessageHandler = ({
                 rejection_reason: RejectionReasonEnum.USER_NOT_FOUND,
                 status: RejectedMessageStatusValueEnum.REJECTED
               }),
+              TE.chain(() =>
+                // setting TTL to 3 years for message-status entries
+                lMessageStatusModel.updateTTLForAllVersions(
+                  [newMessageWithoutContent.id],
+                  94670856 as NonNegativeInteger
+                )
+              ),
+              TE.chain(() =>
+                // setting TTL to 3 years for message entry
+                lMessageModel.patch(
+                  [
+                    newMessageWithoutContent.id,
+                    newMessageWithoutContent.fiscalCode
+                  ],
+                  { ttl: 94670856 } as Partial<MessageWithoutContent>
+                )
+              ),
               TE.getOrElse(e => {
```

Comment:
This error managmenet is not enough: if the update ttl or the patch fails, it will simple throw a not verbose error.
I think we need a specific ai track to specific manage those cases.